process/smartContract/vmInput.go

```diff
@@ -71,6 +71,7 @@ func (sc *scProcessor) createVMCallInput(tx data.TransactionHandler) (*vmcommon.
 	vmCallInput.CallType = callType
 	vmCallInput.RecipientAddr = tx.GetRcvAddr()
 	vmCallInput.Function = function
+	vmCallInput.OriginalTxHash = txHash
 
```

Comment:
Please set originaltxhash and currentxhash as well.

Reply:
Done.